debian/k8s-cni/99_bridge.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "cniVersion": "0.1.0",
+  "name": "kubenet",
```

Comment:
Ah, then my assumption was right (that it didn't contain an overlay)

Seems like I misinterpreted your previous comment then (it clearly would have been to good to be true if kubenet did provide an overlay and worked everywhere)

Converting it into a CNI plugin SGTM